deployment/environment/memory/environment.go

```diff
@@ -160,11 +166,24 @@ func generateMemoryChainSol(inputs map[uint64]SolanaChain) map[uint64]deployment
 	return chains
 }
 
+func generateMemoryChainAptos(inputs map[uint64]AptosChain) map[uint64]deployment.AptosChain {
+	chains := make(map[uint64]deployment.AptosChain)
+	for cid, chain := range inputs {
+		chain := chain
+		chains[cid] = deployment.AptosChain{
+			Client:         chain.Client,
+			DeployerSigner: chain.DeployerKey,
+		}
+	}
+	return chains
+}
+
 func NewNodes(
 	t *testing.T,
 	logLevel zapcore.Level,
 	chains map[uint64]deployment.Chain,
 	solChains map[uint64]deployment.SolChain,
+	aptosChains map[uint64]deployment.AptosChain,
 	numNodes,
 	numBootstraps int,
 	registryConfig deployment.CapabilityRegistryConfig,
```

Comment:
This is a really bad pattern. At least the chain inputs should be a struct. This is not extensible, adding other chain families will become harder every time.